lib/operations/admin_ops.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const executeCommand = require('./db_ops').executeCommand;
+const executeDbAdminCommand = require('./db_ops').executeDbAdminCommand;
+const toError = require('../utils').toError;
+
+/**
+ * Get ReplicaSet status
+ *
+ * @param {Admin} a collection instance.
+ * @param {Object} [options] Optional settings. See Admin.prototype.replSetGetStatus for a list of options.
+ * @param {Admin~resultCallback} [callback] The command result callback.
+ */
+function replSetGetStatus(admin, options, callback) {
+  executeDbAdminCommand(admin.s.db, { replSetGetStatus: 1 }, options, (err, doc) => {
+    if (err == null && doc.ok === 1) return callback(null, doc);
+    if (err) return callback(err, false);
+    callback(toError(doc), false);
+  });
+}
+
+/**
+ * Retrieve this db's server status.
+ *
+ * @param {Admin} a collection instance.
+ * @param {Object} [options] Optional settings. See Admin.prototype.serverStatus for a list of options.
+ * @param {Admin~resultCallback} [callback] The command result callback
+ */
+function serverStatus(admin, options, callback) {
+  executeDbAdminCommand(admin.s.db, { serverStatus: 1 }, options, (err, doc) => {
+    if (err == null && doc.ok === 1) {
+      callback(null, doc);
```

Comment:
this is relatively nitpicky, but can we rewrite this to be consistent with the style above? Upon second glance it looks to me like a number of these methods in here can be refactored to call a common function since they are just changing the command document sent